plugins/formal-verification/formver.annotations/src/org/jetbrains/kotlin/formver/plugin/Builtins.kt

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright 2010-2023 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver.plugin
+
+/**
+ * Built-in function used to verify a given boolean predicate with Viper.
+ */
```

Comment:
Please make it clear that this function is meant merely as a hook for our plugin; it doesn't do verification, it marks something as needing to be verified.